gen-apidocs/generators/api/operation.go

```diff
@@ -174,7 +174,7 @@ func (o *Operation) GetGroupVersionKindSub() (string, string, string, string) {
 // initExample reads the example config for an operation
 func (o *Operation) initExample(config *Config) {
 	path := o.Type.Name + ".yaml"
-	path = filepath.Join(*ConfigDir, config.ExampleLocation, o.Definition.Name, path)
+	path = filepath.Join(ConfigDir, config.ExampleLocation, o.Definition.Name, path)
 	path = strings.Replace(path, " ", "_", -1)
```

Comment:
nit: This seems confusing, `config.ExampleLocation`.
Instead create a static string to represent the `examples` sub dir under `config`.
Or is there a reason to set `example_location: examples` in config.yaml?
Also:
- See line 103 in config.go
- See line 281, definition.go

Reply:
Also, double check the variables for `includes` and `static_includes` directories in `writer.go`.

Reply:
No comment to the `examples` setting. It is legacy code I didn't care to fix.

`static_includes` is retrieved from a dedicated function, `getStaticIncludesDir`, we can easily kill it.